RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -869,18 +869,10 @@ string reverse(string s)
 	return ret;
 }
 
-boolean forbidFamChange()
+boolean is100FamRun()
 {
-	// Answers the question "am I not allowed to change my familiar?"
-	// Returns true for paths with no familiars
-	if(get_property("auto_100familiar") == $familiar[Egg Benedict])
-	{
-		if(have_familiar($familiar[Mosquito]))
-		{
-			return false;
-		}
-	}
-
+	// answers the question of "is this a 100% familiar run"
+	
 	if(get_property("auto_100familiar") == $familiar[none])
```

Comment:
either the left side of this needs a `.to_familiar()` or the right side needs a `.to_string()`. I'd prefer the former.

Reply:
fixed at commit be16e90

Also, does the line below where it checks if it is ""... in that line I think it should not have .to_familiar() but I am not 100% sure. can i get your input on it?

Reply:
jeparo told me to_familiar() already converts "" into $familiar[none] and I actually went ahead and tested it and it is as he says. removed the redundant check at commit 
e1135fa